com.microsoft.mrtk.graphicstools.unity/Runtime/Materials/BlitMaterial.mat

```diff
@@ -0,0 +1,37 @@
+%YAML 1.1
```

Comment:
Can you move this material into the Experimental>Magnifier>Materials folder? That way all magnifier assets and contained together within the same folder (the Acrylic folder is a good example to base off of).